media-proxy/src/api_server_tcp.cc

```diff
@@ -1,19 +1,15 @@
 /*
  * SPDX-FileCopyrightText: Copyright (c) 2023 Intel Corporation
```

Comment:
Please update Copyright (c) 2023 Intel Corporation to Copyright (c) 2024 Intel Corporation